cmd/buildkitd/main_containerd_worker.go

```diff
@@ -52,10 +53,17 @@ func containerdWorkerInitializer(c *cli.Context, common workerInitializerOpt) ([
 	if err != nil {
 		return nil, err
 	}
-	opt, err := containerd.NewWorkerOpt(common.root, socket, ctd.DefaultSnapshotter, labels)
+	workerNet := c.GlobalString("worker-net")
```

Comment:
containerd-worker-net

Reply:
Atually, changing name from ``worker-net`` to ``oci-worker-net`` and ``containerd-worket-net`` make this logic reduendent.

Right now the flag is global flag, and will work for each worker type. By changing name, every worker type implementation has to define its own ``<worker-type>-worker-net``.

I this this change is not required and we should revert the previous change too. ie. keep ``worker-net`` as global.

Reply:
WDYT about having both `<workername>-worker-net` and `worker-net`?
The former one is preferred over the latter one if set.

@kunalkushwaha @tonistiigi